r_gl.cpp

```diff
@@ -5,674 +5,685 @@
 #include <glad/glad.h>
 
 #include "imgui.h"
-#include <imgui/backends/imgui_impl_sdl2.h>
-#include <imgui/backends/imgui_impl_opengl3.h>
 #include <SDL2/SDL_egl.h>
+#include <imgui/backends/imgui_impl_opengl3.h>
+#include <imgui/backends/imgui_impl_sdl2.h>
 
 #define GLM_FORCE_RADIANS
-#include "dependencies/glm/glm.hpp"
 #include "dependencies/glm/ext.hpp"
+#include "dependencies/glm/glm.hpp"
 
-#include "r_common.h"
+#include "input.h"
 #include "platform.h"
-#include "r_itexture.h"
+#include "r_common.h"
 #include "r_gl_batch.h"
 #include "r_gl_texture.h"
 #include "r_gl_texture_mgr.h"
-#include "input.h" 
+#include "r_itexture.h"
 
 const int WINDOW_WIDTH = 1920;
 const int WINDOW_HEIGHT = 1080;
 
-void GLRender::Shutdown(void)
-{
-    // Deinit ImGui
```

Comment:
It changed the sort order of the function declarations? Why?

Reply:
probably to be aligned with the header file

Reply:
Ah, that would make sense.

Reply:
it didn't change the order. it was a leftover from another test...